src/sections/Community/Handbook/projects.js

```diff
@@ -16,14 +16,39 @@ const Maintainer= () => {
           <div className= "content">
             <p>Here are the Layer5 Projects:  </p>
         
-            <h3>1. Layer5</h3>
-            <p>It is the official website for the Layer5 community showing an overview of the Layer5 Projects, the different service mesh landscapes, resources to learn about service Mesh and communities.</p>
-            <h3>2. Meshery</h3>
-            <p>It is a multi-service mesh management plane offering lifecycle management of more types of service meshes than any other tool available today. Meshery facilitates adopting, configuring, operating, and managing the performance of different service meshes and incorporates the collection and display of metrics from applications running on top of any service mesh. </p>
-            <h3>3. Service Mesh Performance</h3>
-            <p>The Service Mesh Performance (SMP) is a vendor-neutral specification for capturing details of environment and infrastructure details, service mesh and it’s configuration, service/application details, and bundling of statistical analysis of the result.</p>
-            <h3>4. GetNightHawk</h3>
-            <p>This Is a Layer 7 (HTTP/HTTPS/HTTP2) performance characterization tool.      Nighthawk is Envoy’s load generator and is written in C++. Meshery integrates Nighthawk as one of (currently) three choices of load generator for characterizing and managing the performance of service meshes and their workloads.</p>
+            <h3><a href="https://layer5.io/">1. Layer5</a></h3>
+            <p>Its service mesh community represents the largest collection of service Mesh projects.Emerging projects like GetNighthawk , community with  MeshMates, catch-all org, “home base”.<br/> Github link: <a href="https://github.com/layer5io">https://github.com/layer5io</a>
```

Comment:
Instead of `Github link: <a href="https://github.com/layer5io">https://github.com/layer5io</a>` can you try adding the github icon and linking that to the respective repositories for all the instances.

Reply:
I think that would not look good

Reply:
@warunicorn19 views?

Reply:
Yes @debo19 , I think the logo there would look much than having the words "GitHub link", @moazzam07 try using the GitHub logo like on this page https://layer5.io/community/handbook/repository-overview, and add the hyperlink to that logo, instead of having a huge link on the page.